metricflow-semantics/metricflow_semantics/mf_logging/pretty_print.py

```diff
@@ -36,7 +36,8 @@ def __init__(
 
     @staticmethod
     def _is_pydantic_base_model(obj: Any):  # type:ignore
-        return isinstance(obj, BaseModel)
+        # Checking the attribute as the BaseModel check fails for newer version of Pydantic.
+        return isinstance(obj, BaseModel) or hasattr(obj, "__config__")
```

Comment:
That's risky but I guess it's ok in this context.

Reply:
I couldn't figure out a good solution after a few minutes of searching, but yeah, I'm looking for a better solution.